content/pages/mirrors-core-latest-redir.md

```diff
@@ -1,7 +1,7 @@
 Title:
-URL:
+URL: mirrors-core-latest-redir.html
```

Comment:
I believe since https://issues.apache.org/jira/browse/LUCENE-8820 the latest-redir is not in use anywhere?